specification/contosowidgetmanager/Contoso.WidgetManager/main.tsp

```diff
@@ -9,21 +9,34 @@ using TypeSpec.Rest;
 using TypeSpec.Versioning;
 using Azure.Core;
 
+@useAuth(OAuth2Auth<[ContosoAuthFlow]>)
 @service({
   title: "Contoso Widget Manager",
 })
 @versioned(Contoso.WidgetManager.Versions)
+
 namespace Azure.Contoso.WidgetManager;
 
-#suppress "@azure-tools/typespec-azure-core/documentation-required" "https://github.com/Azure/typespec-azure/issues/3107"
```

Comment:
@raych1: Why did you remove this suppression and add the doc comments?  Did you see this is a bug in `typespec-azure` we are waiting for them to fix?

Reply:
We should definitely not put suppressions into our main example. We should take the doc fix.

Reply:
@markcowl: I believe you recommended the opposite, we should suppress this until https://github.com/Azure/typespec-azure/issues/3107 is fixed.

Reply:
Documenting versioning enums is, at best, annoying (but no more annoying than literally every other document nag linter we have). Normalizing the suppression of linter rules in our flagship example does far more harm than good, IMO.